px-auto-filter-field.html

```diff
@@ -85,45 +85,27 @@
             },
 
             value: {
-              type: String
+              type: String,
+              observer: '_onChange'
             }
           };
         }
 
         ready() {
           super.ready();
-          const input = this.shadowRoot.querySelector('input');
```

Comment:
I'm confused, as PR title states you're adding tests, not changing `px-auto-filter-field` component. Are changes in `px-auto-filter-field.html` intentional? If so, you need to move them to separated PR or at least to separated commit(s).

Reply:
Ok, this is fine, titles to be improvised in the future